src/scores/stats/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Import the functions from the implementations into the public API
+"""
+
+from .confidence_intervals_impl import dm_test_stats
```

Comment:
I would suggest moving this in the namespace to scores.stats.tests.diebold_mariano. I think we should have a brief discussion to confirm the best way to place this in the namespace.

Reply:
Renamed